Chapter 1. Basics

yosangwon opened the pull request:
This PR is made for comparison and making comments.

yosangwon committed:
First Draft of the first chapter.

Dalgona commented on ko/lessons/basics/basics.md:
```diff
@@ -1,35 +1,35 @@
+### Elixir 설치하기
 
-Installation instructions for each OS can be found on Elixir-lang.org in the [Installing Elixir](http://elixir-lang.org/install.html) guide.
+Elixir-lang.org 홈페이지의 [Elixir 설치 가이드](http://elixir-lang.org/install.html)에서 각 운영체제별로 설치하는 방법을 알아볼 수 있습니다.
```
_note:_ 제 경우에는 링크 텍스트는 원문의 것을 유지했습니다. (importance: low+)

yosangwon replied:
@dalgona 아직 한국어로 번역되지 않은 문서니까 그렇게 하는 쪽이 좋겠네요. 그렇게 할게요.

Dalgona commented on ko/lessons/basics/basics.md:
```diff
@@ -49,9 +49,9 @@ iex> 0x1F
+### 부동 소수점
 
-In Elixir, float numbers require a decimal after at least one digit; they have 64 bit double precision and support `e` for exponent numbers:
+Elixir에서는 64비트 정밀도(double precision)로써 부동 소수점 숫자를 처리하고, `e`를 사용하여 10의 지수를 표현할 수도 있습니다. 부동 소수점 숫자를 표현할 때에는 소수점 앞뒤로 숫자가 한 개 이상 필요합니다.
```
_suggestion:_ double precision: 배정도 (importance: low)

yosangwon replied:
@dalgona "배정밀도"로 옮기는 것도 괜찮을 것 같아요. "배정도"는 사람 이름 같네요.. 괄호로 원문 적어놓은 건 생략하는 게 좋을까요?

Dalgona replied:
@devleoper 
1. 지금까지 "단정도/배정도"만 봐 왔고 "배정밀도"는 본 적이 없어서 구글에 검색해 봤는데 용례가 꽤 나오네요. "배정밀도"라고 하는것도 괜찮을것같아요. 
2. 네 이제 배정밀도가 double precision을 직접 의미하는 거니까 괄호는 없애도 될 것 같네요

Dalgona commented on ko/lessons/basics/basics.md:
````diff
@@ -164,7 +164,7 @@ iex> !false
 ```
 
-There are three additional operators whose first argument _must_ be a boolean (`true` and `false`):
+반면 `and`와 `or`, `not`은 **반드시** 첫번째 인수가 불리언 값(`true`나 `false`)이어야 합니다.
````
강조 포맷이 달라도 괜찮.....겠죠?

yosangwon replied:
@dalgona 한국어에서는 강조할 때 _이탤릭체_를 일반적으로 사용하지 않아요. 글꼴을 배포할 때도 따로 이탤릭체를 만들어 배포하는 경우는 극소수고, 대부분의 운영체제에서 단순히 글자를 기울이는 효과를 주는 경우가 많아요. 또, 일부 운영체제의 경우 이탤릭체에 해당하는 글꼴이 없으면 효과를 주지 않기 때문에 보통 한국어에서 이탤릭체 사용은 지양하는 편이에요.

Dalgona commented on ko/lessons/basics/basics.md:
````diff
@@ -117,11 +117,11 @@ iex> "foo\nbar"
 ```
 
-## Basic Operations
+## 기본 연산
````
개인적으로는 하위 섹션 처음 세개의 헤딩을 "산술 연산", "논리 연산", "비교 연산"이라고 하고싶은데 어떻게 생각하세요?

yosangwon replied:
@dalgona 하긴 "불리언 대수 연산자"는 너무 말이 길어진 것 같네요. "수치 연산자"라는 말도 다시 읽어보니 그닥 와닿지는 않는 것 같고요. 그렇지만 "연산"보다는 "연산자"라는 말을 그대로 두는 쪽이 조금 더 내용을 잘 드러내는 것 같아요. (가령, `+`는 연산자일 뿐이지 연산 자체는 아니니까요.)

말을 해 보고 나니 9번째 줄과 이 라인에도 "연산자"로 바꾸는 편이 좋겠어요.

yosangwon replied:
@Dalgona 타입에서 불리언 대수에 대한 언급이 있어서 여기서도 불리언 대수 연산이라고 언급하기는 했어요. Boolean 타입은 그대로 불리언 대수로 둬도 되겠죠?

Dalgona replied:
@devleoper 논리형/논리값/논리 연산은 어때요? 모호하거나 아귀에 안 맞는 점이 있을까요?

yosangwon replied:
@dalgona 논리값/논리 연산은 괜찮은 것 같아요. "논리형(type)"은 "논리형 프로그래밍 언어" 외에는 용례가 적네요.

yosangwon committed:
Changed by received feedbacks

 - Link to "Installing Elixir" is now English, because the link has not
   been translated to Korean.
 - changed "64비트 정밀도" to "배정밀도" which is shorter them used mo-
   re widely.

yosangwon committed:
Changed some stuffs for consistency:

 - Boolean (as type): 부울 값
 - Boolean operator/operation: 논리 연산자/연산

yosangwon committed:
Fixed the Long-term typo
